yaki_admin/src/features/captain/components/teamMate.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="teamMate">
+    <div class="teamMateAvatarInfo">
+      <img class="avatar" src="../../../assets/avatar.png" alt="Avatar">
+      <div class="teamMateInfo">
+        <h1 class="name">{{ teamMate.firstName }} {{ teamMate.lastName }}</h1>
+        <h2 class="email">{{ teamMate.email }}</h2>
+      </div>
+    </div>
+    <div class="deleteEditIcon">
+      <img class="teamMateIcon" src="../../../assets/Edit.png" alt="">
+      <img class="teamMateIcon" src="../../../assets/Delete.png" alt="">
+    </div>
+  </div>
+</template>
+  
+<script lang="ts">
+import { defineComponent } from 'vue';
+import type { TeamMateType } from '../../../services/teamMate.type';
+export default defineComponent({
+  name: 'TeamMate',
+  props: {
+    // The prop 'teamMate' is of type 'TeamMateType' and is required.
+    teamMate: {
+      type: Object as () => TeamMateType,
```

Comment:
Object as TeamMateType 
work fine for props type, anonymous function isn't needed.